.github/workflows/ci.yml

```
@@ -152,8 +164,17 @@ jobs:
       - name: Generate build summary
         if: always()
         run: |
```

actionlint (1.7.8)

property "job-start" is not defined in object type {}

(expression)